package.json

```diff
@@ -40,6 +40,20 @@
         "category": "VOOD"
       }
     ],
+    "configuration": {
+      "title": "Visual OO Debugger",
+      "properties": {
+        "visual-oo-debugger.preferredView": {
+          "description": "Specifies the how the debug data will be visualised.",
+          "type": "string",
+          "default": "vis.js",
+          "enum": [
+            "vis.js",
+            "mock.js"
```

Comment:
I can remove the entry, but I need a PanelViewProxy class other then visjsPanelView for the getPanelViewByConfiguration function in extension.ts. Otherwise, the function would be just:
return new VisjsPanelView(this.context);

Reply:
In the getPanelViewByConfiguration function add a switch case with (for now) only one case ("vis.js"). Then remove everything mock related.

Reply:
Done as suggested -> resolved